docs/POOL_EXAMPLES_ETH.md

```diff
@@ -1,8 +1,39 @@
 # Pool Examples for ETH
 
-This is a collection of examples how to connect ethminer to your favorite ETH pool (alphabetic order).
+Pool connection definition is issued via `-P` argument which has this syntax
 
-* Stratum connection is preferred than getwork connection due to its better network latency.
+```
+-P scheme://user[.workername][:password]@hostname:port[/...]
+```
+__values in square brackets are optional__
+
+where `scheme` can be any of:
+
+* `http` for getwork mode (geth)
+* `stratum+tcp` for plain stratum mode
+* `stratum1+tcp` for plain stratum eth-proxy compatible mode
+* `stratum2+tcp` for plain stratum NiceHash compatible mode
+
+## Secure socket comunications for stratum only
+
+Ethminer supports secure socket communications (where pool implements and offers it) to avoid the risk of a [man-in-the-middle attack](https://en.wikipedia.org/wiki/Man-in-the-middle_attack)
+To enable it simply replace tcp with either:
+
+* `ssl` to enable secure socket communication
+* `tls12` to enable secure socket communication **with strong TLS12** checking
+
```

Comment:
THIS IS NOT TRUE!!

https://github.com/ethereum-mining/ethminer/blob/master/libpoolprotocols/PoolURI.cpp#L43

please document only ``ssl``  and ``tls`` ..

ssl = tls1.2 , allow only "strong" TLS 1.2
tls = all tls versions allowed (compatibility)

So + SSL would be the recommended way to use.

Reply:
Please do not shout ... trying to do at my best not here to be reproached.

Reply:
sorry wasn't meant that way. :(

Reply:
Now I'm confused. Aren't ssl and tls different protocols, tls being the successor of the deprecated ssl? Some ports will still allow the negotiation of an ssl connection.

It just seems odd to and old man like me to use ssl to indicate a tls1.2 requirement!

Reply:
Agree but all those are, imho, to be considered old backward compatible aliases which, BTW, where not depicting reality the proper way.

God save us if we have to ask users to change their precious .bat files.

My hope is that stratum[s|ss] may help amend this situation with time.

Reply:
I don't have a strong opinion about all of this, but I'll certainly continue using the '+' notations. Generally though, the proliferation of options can lead to more confusion.

Reply:
I have a conflicting view about that.

On one side we have users who NEVER RTFM and go cut and paste. If we can help spreading docs with a single syntax which includes auto-stratum detection we have solved many issues.

On the other side agree with you `+` notations bring you straight to **what you want** : say you want to test a particular flavour on a pool ... auto-detection might not help.

Reply:
With the pools I use auto-detect has not affected me negatively, even if it chooses proxy mode over stratum. Pool response time is the same.